src/javascript/app/pages/cashier/cashier.js

```diff
@@ -43,6 +43,19 @@ const Cashier = (() => {
         });
     };
 
+    const showP2PNote = async () => {
+        const is_agent = !(await BinarySocket.send({ p2p_agent_info: 1 })).error;
+        const has_buy = await checkP2PHasOffer('buy');
+        const has_sell = await checkP2PHasOffer('sell');
```

Comment:
same here, we should check if has_buy first so we don't make extra calls if it's not needed